webrtc.html

```diff
@@ -2400,14 +2409,25 @@
 
         <dl class="idl" title=
           "interface RTCRtpReceiver">
-        <dt>readonly attribute MediaStreamTrack track</dt>
+          <dt>readonly attribute MediaStreamTrack track</dt>
 
-        <dd>
-          <p>The <dfn id=
-          "dom-rtpreceiver-track"><code>RTCRtpReceiver.track</code></dfn>
-          attribute is the track that is immutably associated with this
-          <code><a>RTCRtpReceiver</a></code> object.</p>
-        </dd>
+          <dd>
+            <p>The <dfn id=
+            "dom-rtpreceiver-track"><code>RTCRtpReceiver.track</code></dfn>
+            attribute is the track that is immutably associated with this
+            <code><a>RTCRtpReceiver</a></code> object.</p>
+          </dd>
+
+          <dt>readonly attribute DOMString mid</dt>
+          <dd>
+            The <dfn id="dom-rtpreceiver-mid"><code>RTCRtpReceiver.mid</code></dfn>
+            attribute is the value of the a=mid SDP attribute that is immutably
+            associated, via <code>setRemoteDescription</code>, with this
+            <code><a>RTCRtpReceiver</a></code> object.
+            In the case where no a=mid attribute is present in the remote
+            description, a random value will be generated.
```

Comment:
yes - a stable value, so that the application could still use .mid to tell its receivers apart. 

Probably should say "unique value" or "unique random value"